cmd/twins/main.go

```diff
@@ -22,8 +22,10 @@ import (
 	mqtt "github.com/eclipse/paho.mqtt.golang"
 	"github.com/mainflux/mainflux"
 	"github.com/mainflux/mainflux/logger"
+	"github.com/mainflux/mainflux/normalizer"
```

Comment:
No need for Noramlizer at all. Normalizer is a service that subscribes to NATS and reads messages which are "unnormalized" SenML (by the standard SenML can be an array of JSON structures, called "unnromalized", and can parse SenML content and transform it to "normalized" - by breaking this array in individual messages which is shoots back to NATS.

Writers cn read only "normalized" SenML - i.e. they know to read only individual messages, not an array, so they write these messages one by one into DB.

Normalizer is a common procedure for all Writers, so instaed of putting SenML normalization procedure in each writer, this is separates into individual service.

Twin service will not need to normalize anything, as messages (content) comming over Twin state channel will be in the form of Twin state - i.e. arbitrary JSON that represents new state - and should be written "tel quel" into the Twin's Mongo DB.